src/styles/settings/_variables.css

```
@@ -0,0 +1,32 @@
+/* Colors */
+:root{
+  --light-yellow: #ffe21d;
+  --dark-yellow: #ffd21d;
+  --light-blue: #24afff;
+  --dark-blue: #248bff;
+  --salmon: #f26c6c;
+  --green: #27b667;
+}
+
+/* Fonts */
+:root{
+  --default-font: 'Montserrat', sans-serif;
+  --title-font-size: 30px;
+  --text-font-size: 16px;
+}
+
+/* Spaces */
+:root{
+  --space-unit: 10px;
+
+  --margin: calc(0.5 * var(--space-unit));
+  --padding: calc(0.5 * var(--space-unit));
+
+  --margin-small: var(--margin);
+  --margin-medium: calc(2 * var(--margin));
+  --margin-large: calc(4 * var(--margin));
+
+  --padding-small: var(--padding);
+  --padding-medium: calc(2 * var(--padding));
+  --padding-large: calc(4 * var(--padding));
```

Muito bom, tu pode até usar as mesmas variáveis tanto pra padding quanto pra margens, não precisa ter variáveis diferentes pra cada uma, então ficar algo tipo:

```suggestion
  --space-unit: 10px;

  --space-default: calc(0.5 * var(--space-unit));
  
  --space-small: var(--margin);
  --space-medium: calc(2 * var(--margin));
  --space-large: calc(4 * var(--margin));
```

Também não precisa fazer um seletor `:root` pra cada um, pode ser tudo no mesmo seletor.